openbas-api/src/main/java/io/openbas/xtmhub/collector/XtmHubCollectorConfig.java

```diff
@@ -0,0 +1,18 @@
+package io.openbas.xtmhub.collector;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Setter
+@Component
+@ConfigurationProperties(prefix = "openbas.xtm.hub.collector")
+public class XtmHubCollectorConfig {
+  @Getter private boolean enable;
+
+  @Getter @NotBlank private String id;
+
+  @Getter @NotBlank private Integer connectivityCheckInterval = 60 * 60 * 1000;
```

Comment:
This feels like magic numbers. Can you comment on what this really represents (i mean, it's pretty obvious but none the less, but just a comment like "default to 1h")